src/shell/commands/data_operations.cpp

```diff
@@ -2119,10 +2125,15 @@ bool count_data(command_executor *e, shell_context *sc, arguments args)
         int option_index = 0;
         int c;
         c = getopt_long(
-            args.argc, args.argv, "p:b:t:h:x:s:y:v:z:dan:r:", long_options, &option_index);
+            args.argc, args.argv, "cp:b:t:h:x:s:y:v:z:dan:r:", long_options, &option_index);
         if (c == -1)
             break;
+        // if input [-p|--partition] etc. means you want to get precise count by scanning.
```

Comment:
```suggestion
        // if specifying [-p|--partition] or [-c|--precise] means you want to get precise count by scanning.
```

Reply:
Inputting any valid parameter will means want precise result, not only  `[-p|--partition] or [-c|--precise]`